inc/functions_user.php

```diff
@@ -123,12 +123,7 @@ function validate_password_from_uid($uid, $password, $user = array())
 }
 
 /**
- * Updates a user's password.
- *
- * @param int The user's id.
- * @param string The md5()'ed password.
- * @param string (Optional) The salt of the user.
- * @return array The new password.
+ * @deprecated deprecated since version 1.8.5 Please use other alternatives.
```

Comment:
Though it's deprecated the rest shouldn't be removed imho. Also `1.8.6` would be correct here too.